src/e2e/pages/landingPage.js

```diff
@@ -25,6 +28,18 @@ export class LandingPage {
     this.falseBannerCloseButton = page.locator('#close-button')
     this.scanEmailAddressInput = page.locator('#scan-email-address')
     this.checkForBreachesButton =  page.getByRole('button', {name: 'Check for breaches'})
+
+    // hero section
```

Comment:
Just food for thought: with all these selectors, is this page object even useful? Doesn't seem like they get re-used often, in which case it's just another layer of indirection.